### PR TITLE
Created Configuration file for hi localization

### PR DESCRIPTION
**Pull request by kumarankit999**
Localized Configuration file for hi localization

### CODE REVIEW

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html
Comment (sftim) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:1]: No need to add this file. Instead, the simplest thing to do is to omit it and not link to it.

Comment (kumarankit999) [reply content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:1]: OK I do it soon

**Comment by utkarsh-singh1** [comment]
Hi @kumarankit999, This issue is already in working - https://github.com/kubernetes/website/issues/41496.

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## {{% heading "prerequisites" %}}
+
+### Creating Kubernetes ConfigMaps & Secrets
+
+कुबेरनेट्स में डॉकर कंटेनर के लिए पर्यावरण चर सेट करने के कई तरीके हैं,
+इसमें शामिल हैं: Dockerfile, kubernetes.yml, Kubernetes configMaps, और Kubernetes Secrets। में
+ट्यूटोरियल, आप सीखेंगे कि अपने पर्यावरण चर सेट करने के लिए बाद वाले दो का उपयोग कैसे करें
+मान आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स का उपयोग करने के लाभों में से एक और
+रहस्य यह है कि इन्हें असाइन किए जाने सहित कई कंटेनरों में पुन: उपयोग किया जा सकता है
```
Comment (kundan2707) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:22]:
```suggestion
ट्यूटोरियल, आप सीखेंगे कि अपने पर्यावरण चर सेट करने के लिए बाद वाले दो का उपयोग कैसे करें
जिसका मान आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने के लाभों में से एक यह है की उन्हें कई कंटेनरों में पुन: उपयोग किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग पर्यावरण चर को निर्धारण करना भी शामिल है।

```

**Comment by Ritikaa96** [comment]
Hi @kumarankit999 are you willing to address the reviews suggested? still working on this PR?

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<html lang="en">
+
+<body>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+        <div>
+            <h2 class="katacoda-gone">
+                सामग्री अनुपलब्ध
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:22]:
```suggestion
                कैटाकोडा अनुपलब्ध

```

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            </h2>
+            <p>
+                क्लस्टर बनाने के लिए इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
+                अधिक जानकारी के लिए देखें
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:26]:
```suggestion
                अधिक जानकारी के लिए <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">शटडाउन घोषणा देखें।</a>

```

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों बाहरी बनाना है।
+विशेष रूप से, आप सीखेंगे कि पर्यावरण सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स का उपयोग कैसे करें
+चर और फिर माइक्रोप्रोफ़ाइल कॉन्फ़िगरेशन का उपयोग करके उनका उपभोग करें।
+
+
+## {{% heading "prerequisites" %}}
+
+### Creating Kubernetes ConfigMaps & Secrets
+
+कुबेरनेट्स में डॉकर कंटेनर के लिए पर्यावरण चर सेट करने के कई तरीके हैं,
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:18]: Update the passage with better translation.

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+इसमें शामिल हैं: Dockerfile, kubernetes.yml, Kubernetes configMaps, और Kubernetes Secrets। में
+ट्यूटोरियल, आप सीखेंगे कि अपने पर्यावरण चर सेट करने के लिए बाद वाले दो का उपयोग कैसे करें
+जिसका मान आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने के लाभों में से एक यह है की उन्हें कई कंटेनरों में पुन: उपयोग किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग पर्यावरण चर को निर्धारण करना भी शामिल है।
+
+विभिन्न कंटेनरों के लिए अलग-अलग पर्यावरण चर।
+
+कॉन्फिगमैप्स एपीआई ऑब्जेक्ट हैं जो गैर-गोपनीय कुंजी-मूल्य जोड़े को संग्रहीत करते हैं। इंटरैक्टिव में
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:25]:
```suggestion
कॉन्फिगमैप्स एपीआई ऑब्जेक्ट हैं जो गैर-गोपनीय कुंजी-मूल्य जोड़े को संग्रहीत करते हैं। इंटरैक्टिव

```

File: content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+ट्यूटोरियल में आप सीखेंगे कि एप्लिकेशन के नाम को संग्रहीत करने के लिए कॉन्फिग मैप का उपयोग कैसे करें। अधिक जानकारी के लिए
+कॉन्फ़िगमैप्स के संबंध में जानकारी, आप दस्तावेज़ पा सकते हैं
+[यहाँ](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+
+हालाँकि सीक्रेट्स का उपयोग कुंजी-मूल्य जोड़े को संग्रहीत करने के लिए भी किया जाता है, लेकिन वे कॉन्फ़िगमैप्स से भिन्न होते हैं
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:30]: Update the passage.

File: content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+reviewers:
+- eparis
+- pmorie
+title: Configmap का उपयोग करके रेडिस को कॉन्फ़िगर करना
+content_type: tutorial
+---
+
+<!-- overview -->
+
+यह पृष्ठ कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करने का वास्तविक विश्व उदाहरण प्रदान करता है और इसका निर्माण करता है [उपयोग करने के लिए पॉड को कॉन्फ़िगर करें ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) काम.
```
Comment (utkarsh-singh1) [inline content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md:11]: Update the passage with better translation and wrap the passage.

**Comment by utkarsh-singh1** [comment]
@kumarankit999, Changes in #46528 had rebase this PR.

**Comment by utkarsh-singh1** [comment]
You can use this [link](https://github.com/kubernetes/website/pull/46528) to rebase your PR.

**Pull request closed without merging**